joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstCreator.scala

```diff
@@ -385,6 +386,66 @@ class AstCreator(
       Seq(Ast())
   }
 
+  def astForProcDefinitionContext(ctx: ProcDefinitionContext): Seq[Ast] = {
+    if (ctx == null) return Seq(Ast())
```

Comment:
This check is not needed. If you get here, `ctx` must have type `ProcDefinitionContext` (coming from `astForPrimaryContext`)